### PR TITLE
Add Document Preview page route

### PR DESCRIPTION
**Pull request by subinasr**
- Addresses #2440 and #2442

## Changes
* Add layout to the document preview page
* Add a side pane for the document preview

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/Base/configs/routes.tsx [hunk 1]
```diff
@@ -427,6 +427,16 @@ const groupAssessmentEditRoute = wrap({
     },
 });
 
+const documentPreview = wrap({
+    path: '/document-preview/',
+    title: 'DocumentPreview',
```
Comment (samshara) [inline app/Base/configs/routes.tsx]:
```suggestion
    title: 'Document Preview',
```

File: app/Base/configs/routes.tsx [hunk 2]
```diff
@@ -427,6 +427,16 @@ const groupAssessmentEditRoute = wrap({
...
+    navbarVisibility: true,
+    component: lazy(() => import('#views/Project/Tagging/Export/DocumentPreview')),
+    componentProps: {
+    },
+    visibility: 'is-anything',
```
Comment (samshara) [inline app/Base/configs/routes.tsx:437]: maybe only visible when authenticated

File: app/views/Project/Tagging/Export/DocumentPreview/index.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+    IoDocument,
+    IoDownloadOutline,
+    IoCopyOutline,
+    IoInformationCircleOutline,
+    IoClose,
+} from 'react-icons/io5';
+
```
Comment (samshara) [inline app/views/Project/Tagging/Export/DocumentPreview/index.tsx]: remove this newline

File: app/Base/configs/routes.tsx (hunk 2 above)
Comment (AdityaKhatri) [reply app/Base/configs/routes.tsx:437]: No, this is fine.

**Pull request merged**